testing/regress/ecl/aggidx1.ecl

```diff
@@ -24,6 +24,8 @@ useSequential := #IFDEFINED(root.useSequential, false);
 
 //--- end of version configuration ---
 
+#option ('obfuscateOutput', true);
```

Comment:
I was waiting for this line of reasoning.
Having  a cap on the size of our regression suite (note no plural) makes no sense unless you also want to cap all further production of the src *and* are content that the suite is already complete.
Perhaps we need another and more thorough suite to better tailor their usage.

If we start changing existing tests then we need to be 100% sure that we haven't negated *any* factor of its original purpose.

As you pointed out, and I concur, a runtime test for this fix should be a must, given it's nature, and should be worthy of a new test unto itself. 

Having a individual test per closed jira is almost required until we correctly utilise coverage analysis, such that we can state that an existing test covers that which is required to test (and future test) the given code change.

In this particular case, perhaps I should have said that "aggidx1.ecl should *at least* be copied...". Any other PR that adds a test is subject to the review process, such that aggidx1.ecl now is. Not that there is anything wrong with this by default but there maybe be if this is not as complete as one might expect.